index.html

```diff
@@ -45,5 +45,120 @@ <h2>LET'S CONNECT</h2>
       </ul>
       
     </section>
+    <section class="cardworks-one">
+  <img src="./images/Snapshoot Portfolio.png" alt="" class="snapshoot">
+  <div class="left-blockone"></div>
+  <h3 class="headings">Tonic</h3>
+<img src="./images/Frame 2.png" alt="" class="frame">
+  <p>
+    A daily selection of privately personalized reads; no accounts or sign-ups required.
+  </p>
+  <img src="./images/Tags.png" alt="" class="tag">
+       
+      <div class="buttons">
+       <button class="press">See project</button> 
+      </div>
+</section>
+<section class="cardworks-two">
+  <img src="./images/portfolio.png" alt="" class="snapshoot">
+  <div class="left-blockone"></div>
+  
+   <h4 class="headings">Multi-post stories</h4> 
+  
+  
+<img src="./images/Frame 2.png" alt="" class="frame">
+  <p>
+    A daily selection of privately personalized reads; no accounts or sign-ups required.
+  </p>
+  <img src="./images/Tags.png" alt="" class="tag">
+      <div class="buttons">
+       <button class="press">See project</button> 
+      </div> 
+
+</section>
+<section class="cardworks-three">
+  <img src="./images/Snapshoot Portfolio (3).png" alt="" class="snapshoot">
+  <div class="left-blocktwo"></div>
+  <h5 class="headings">Tonic</h5>
+<img src="./images/Frame 2.png" alt="" class="frame">
+  <p>
+   A daily selection of privately personalized reads; no accounts or sign-ups required.
+  </p>
+ <img src="./images/Tags.png" alt="" class="tag">
+      <div class="buttons">
+      <button class="press">See project</button>  
+      </div>
+</section>
+<section class="cardworks-four">
+  <img src="./images/Snapshoot Portfolio (4).png" alt="" class="snapshoot">
+  <div class="left-blockthree"></div>
+  <h6 class="headings">Multi-Post Stories</h6> 
+  <img src="./images/Frame 2.png" alt="" class="frame">
+  <p>
+  A daily selection of privately personalized reads; no accounts or sign-ups required. 
+  </p>
+  <img src="./images/Tags.png" alt="" class="tag">
+      <div class="buttons">
+       <button class="press">See project</button> 
+      </div>
+</section>
```

Comment:
- [ ] As suggested by the reviewer [here](https://github.com/brendalutiku/Portfolio/pull/2#discussion_r1154242818) please make sure to wrap all cardsworks in one `section` with `display: grid` property and use `display: flex` for individual project cards according to the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md) 🛠 Following is a quote for a related requirement 👇

  > _In order to lay out the elements in "Works" section you should use CSS Grid and Flexbox combined (it is mandatory that you use both). Use CSS Grid in the main container that contains all of the cards and use Flexbox to lay out an individual card._

- [ ] As suggested by the reviewer [here]() please make sure to add some space/padding/margin between headline section and work section 😃

You design before 🔽 | Your design now 🔽 | Figma design 🔽
---| ---| ---|
![image](https://user-images.githubusercontent.com/106119105/229154186-bb6af1a0-a559-4e84-bd01-7cd0c1f1007b.png) | ![image](https://user-images.githubusercontent.com/106119105/229154410-9b245192-91c0-4fd9-86d2-d85a94695d2e.png) | ![image](https://user-images.githubusercontent.com/106119105/229154598-3f271a8d-c205-432d-9d0d-aaca05f4b118.png)

- [ ] Also, please make sure to add some gap/space/padding/margin between individual project cards as given in the Figma design guidelines and also suggested by the reviewer [here](https://github.com/brendalutiku/Portfolio/pull/2#discussion_r1154248069) ✨📌 Please note that the color of background is `#E5E5E5;`

You design before 🔽 | Your design now 🔽 | Figma design 🔽
---| ---| ---|
![image](https://user-images.githubusercontent.com/106119105/229158299-019a8a35-1610-4df8-bc74-1dc654ac0485.png) |![image](https://user-images.githubusercontent.com/106119105/229158450-5c54a746-ed0f-4963-9ca5-f51de8f190eb.png) | ![image](https://user-images.githubusercontent.com/106119105/229158781-1b1cd769-ad51-4b4e-b944-b775b9f08eee.png)


- [OPTIONAL] Good job using only one `h1` per page 🥇 I think it would be better if you use `h2`for `.headings` instead of using various `hx` for `.headings` ✨